File: src/gto.cpp
#include "gto.h"

GTO::GTO(double coeff, Vec3 rr, double alphaa, int ll, int mm, int nn){
    coef = coeff;
    r = rr;
    x = r.getx();
    y = r.gety();
    z = r.getz();
    alpha = alphaa;
    l = ll;
    m = mm; 
    n = nn;
}

double GTO::compute_radius(){
    double tmp = pow(x,2.0) + pow(y,2.0) + pow(z, 2.0);
    return sqrt(tmp);
}

File: python/GTO.py
import numpy as np
from config import Config
cf = Config()

class GTO(object):
    def __init__(self, alpha, coeff) -> None:
        self.alpha = alpha
        self.coeff = coeff

class CGF(object):
    def __init__(self, GTO_list:list) -> None:
        self.size = len(GTO_list)
        self.GTO_list = GTO_list
        pass



